python/paddle/v2/framework/tests/test_fit_a_line.py

```diff
@@ -4,6 +4,7 @@
 import paddle.v2.framework.optimizer as optimizer
 
 from paddle.v2.framework.framework import Program, g_program
+from paddle.v2.framework.save_load import save_persistables, load_persistables
```

Comment:
It seems from this example that what users need is save/load_persistables. Then, do we need save/load_params?

Reply:
Yes, save/load_params used for inference.